packages/@lwc/babel-plugin-component/package.json

```diff
@@ -43,6 +43,8 @@
     },
     "dependencies": {
         "@babel/helper-module-imports": "7.24.7",
+        "@babel/traverse": "^7.25.1",
+        "@babel/types": "^7.25.0",
```

Comment:
`@lwc/babel-plugin-component` only uses these packages for the typescript types, but I guess we need it as a `dep` rather than a `devDep` for the benefit of TS consumers, correct?